packages/common/src/components/logos/Svelte.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+export default ({
```

Comment:
Shouldn't this one come from the [templates repo](https://github.com/codesandbox/codesandbox-templates)? 🤔

Reply:
Same as above